posts/MichelaRocchettiBlogPost/Michela_JSoC.qmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "GSoC '24: IPUMS.jl Small Project"
+description: "A summary of my project for Google Summer of Code"
+author: "Michela Rocchetti"
+date: "8/26/2024"
+toc: true
+engine: julia
+categories:
+  - gsoc
+  - geospatial 
+  - census
+---
+
+# Hello! 👋
+
+Hi! I am Michela, I have a Master's degree in Physics of Complex Systems and I am currently working as a software engineer in Rome, where I am from. 
+During my studies, I became interested in the use of modelling and AI methods to improve healthcare and how these tools can be used to to better understand how cultural and social backgrounds influence the health of individuals. 
+I am also interested in computational modelling of the brain and the human body and its implications for a better understanding of certain pathological conditions. 
+
+With these motivations in mind, I heard about Google Summer of Code. 
+Since I had studied Julia in some courses and given that the language is expanding rapidly, I decided to find a project within Julia. 
+As a result, I found the project of [Jacob Zelko (@TheCedarPrince)](https://jacobzelko.com) to start this experience. 
+
+> For more information about me you could find me here: [**LinkedIn**](https://www.linkedin.com/in/michela-rocchetti-261793218/), [**GitHub**](https://github.com/MichelaRocchetti)
+
+# Project Description 
+
+*IPUMS* is the "world's largest available single database of census microdata", providing survey and census data from around the world. 
+It includes several projects that provide a wide variety of datasets.
+The information and data collected by *IPUMS* are useful for comparative research, as well as for the analysis of individuals in their life contexts.
+These data can be used to create a more comprehensive dataset that will facilitate research on the social determinants of health for different types of diseases, social communities and geographical areas. 
+
+![](./IPUMS_grid_logo.png)
+
+> To learn more about IPUMS, visit the [website](https://www.ipums.org) 
+
+The primary objectives of this proposal is to:
+
+1. Develop a native Julia package to interact with the APIs available around the datasets *IPUMS* provides.
+
+2. Provide useful utilities within this package for manipulating *IPUMS* datasets.
+
+3. Compose this package with the wider Julia ecosystem to enable novel research in health, economics and more. 
+
+# Tasks and Goals
+
+1. Expand some of the functionality developed in `ipumsr`  *IPUMS* NHGIS
+    - Create a link between OpenAPI documentation and the functions internally used in IPUMS.jl:
+    updating already present function, determine if updating is needed and test them
+    - Develop functionality similar to the get_metadata_nghis function present in ipumsr
+
+2. Update *IPUMS* documentation
+    - Set-up and deploy of DocumenterVitepress.jl  
+    - Write a blog post on how IPUMS.jl can compose within the ecosystem.
+
+
+# Workflow 
```

Comment:
Could you rename this to something else since this is more about what you did versus the whole workflow of working with IPUMS.jl?